test/fixtures/expected-output-empty-description-case.html

```diff
@@ -0,0 +1,379 @@
+    font-size: 28px;
+    margin-top: 0;
+  }
+
+  .card--vuln .card__cta p {
+    margin: 0;
+    text-align: right;
+  }
+
+  </style>
+
+  <div class="layout-stacked__header header">
+
+    <header class="project__header">
+      <div class="layout-container--short">
+        <h1 class="project__header__title" style="margin-bottom: 0.5em;">
+          Snyk test report
+        </h1>
+        <p class="timestamp">November 8th 2017, 8:27:48 pm</p>
+        <div class="meta-counts">
+          <div class="meta-count">
+            <span>11</span> <span>known vulnerabilities</span>
+
+          </div>
+          <div class="meta-count"><span>1 vulnerable dependency paths</span></div>
+          <div class="meta-count"><span>140</span> <span>dependencies</span></div>
+        </div><!-- .meta-counts -->
+      </div><!-- .layout-container--short -->
+    </header><!-- .project__header -->
+
+  </div><!-- .layout-stacked__header -->
+
+  <div class="layout-stacked__content">
+
+    <div class="layout-container--short" style="padding-top: 35px;">
+
+      <div class="cards--vuln filter--patch filter--ignore">
+
+        <div class="card card--vuln  disclosure--not-new severity--medium" >
+
+          <h2 class="card__title">MPL-2.0 license</h2>
+
+          <div class="card__section">
+
+            <div class="label label--medium">
+              <span class="label__text">medium severity</span>
+            </div>
+
+            <hr/>
+
+            <ul class="card__meta">
+              <li class="card__meta__item">Module:
+                org.mozilla:rhino
+              </li>
+
+              <li class="card__meta__item">Introduced through:
+
+                  
+
+                  
+                  mytestmodule@0.0.0, com.github.fge:json-schema-validator@2.2.6 and others
+
+
+                </li>
+
+              </ul>
+
+            <hr/>
+
+            <h3 class="card__section__title">Detailed paths</h3>
+
+            <ul class="card__meta__paths">
+              <li>
+                <span class="list-paths__item__introduced"><em>Introduced through</em>: mytestmodule@0.0.0 <span class="list-paths__item__arrow">›</span> com.github.fge:json-schema-validator@2.2.6 <span class="list-paths__item__arrow">›</span> com.github.fge:json-schema-core@1.2.5 <span class="list-paths__item__arrow">›</span> org.mozilla:rhino@1.7R4</span>
+                </li>
+              </li>
+
+            </ul><!-- .list-paths -->
+
+          </div><!-- .card__section -->
+
+          <hr/>
+
+            <!-- Overview -->
+            <p>No description available.</p>
+
+
+          <hr/>
+
+          <div class="cta card__cta">
+            <p><a href="https://snyk.io/vuln/snyk:lic:maven:org.mozilla:rhino:MPL-2.0">More about this vulnerability</a></p>
+          </div>
+
+        </div><!-- .card -->
+
+      </div><!-- cards -->
+
+    </div>
+
+  </div><!-- .layout-container -->
+
+</main><!-- .layout-stacked__content -->
+
+</body>
+</html>
```

Comment:
It doesn't look like this file is used as a test fixture, maybe just remove it? Or am I missing something?